optimum/habana/trl/trainer/ppo_trainer.py

```diff
@@ -539,11 +539,11 @@ def step(
                 active_full_logprobs = logprobs_from_logits(logits_or_none, None, gather=False)
                 ref_full_logprobs = logprobs_from_logits(ref_logits_or_none, None, gather=False)
 
-                rewards, non_score_reward = self.compute_rewards(
+                rewards, non_score_reward, kls = self.compute_rewards(
                     scores, active_full_logprobs, ref_full_logprobs, masks
                 )
             else:
-                rewards, non_score_reward = self.compute_rewards(scores, all_logprobs, ref_logprobs, masks)
+                rewards, non_score_reward, kls = self.compute_rewards(scores, all_logprobs, ref_logprobs, masks)
```

Comment:
@libinta , how was this not caught in tests earlier, dont we have ppo tests?

Reply:
ppo ci is here. https://github.com/huggingface/optimum-habana/pull/1020. after this PR is merged. I need to update 1020, please help merge it then.